doc/topics/windows/windows-package-manager.rst

```diff
@@ -490,8 +487,8 @@ Use :mod:`pkg.list_pkgs <salt.modules.win_pkg.list_pkgs>` to display a list of p
     # From the minion in masterless mode
     salt-call --local pkg.list_pkgs
 
-The above command displays the software name and the version for every package installed
-on the system irrespective of whether it was installed by Salt package manager or not.
+The  command displays the software name and the version for every package installed
```

Comment:
This is a minor thing and doesn't need to prevent merging, but there's an extra space before "command" now.